src/client/components/DirectoryPage/DirectoryHeader/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { FunctionComponent } from 'react'
+import {
+  Typography,
+  createStyles,
+  makeStyles,
+  useTheme,
+  useMediaQuery,
+  Hidden,
+  Button,
+} from '@material-ui/core'
+import { ApplyAppMargins } from '../../AppMargins'
+import GoDirectoryInput from '../../widgets/GoDirectoryInput'
+import { SearchResultsSortOrder } from '../../../../shared/search'
+import useAppMargins from '../../AppMargins/appMargins'
+import BetaTag from '../../widgets/BetaTag'
+import arrow from '../assets/arrow.svg'
+
+type DirectoryHeaderProps = {
+  onQueryChange: (query: string) => void
+  onSortOrderChange: (order: SearchResultsSortOrder) => void
+  onClearQuery: () => void
+  query: string
+  getFile:(queryFile: string) => void
+  getState:(queryState: string) => void
+  getEmail:(queryEmail: string) => void
```

Comment:
i'm seeing inconsistent formatting here and there, leading me to believe that prettier isnt really working correctly for tsx files